daprdocs/content/en/developing-applications/sdks/_index.md

```diff
@@ -35,6 +35,7 @@ The Dapr SDKs are the easiest way for you to get Dapr into your application. Cho
 | [Python]({{< ref python >}}) | In Development | ✔ | [gRPC]({{< ref python-grpc.md >}}) | [FastAPI]({{< ref python-fastapi.md >}})<br />[Flask]({{< ref python-flask.md >}}) |
```

Comment:
Can we put these "In Development" SDK to v1.0 now? Or are you going to do this in another change? If not I would do them in this PR.

Reply:
So you want it changed from "state" to "version"? are they all v1.0? I believe PHP is not.
Maybe for everything that isn't v1.0 we should say "preview" or "planned"

Reply:
@AaronCrawfis - I know you created this page - thoughts on what this should be for v1.0?

Reply:
v1.0 only for Go, .NET, Java and Python. The rest are preview.